src/main/java/net/schmizz/sshj/signature/Signature.java

```diff
@@ -27,10 +27,32 @@
      *
      * @param pubkey (null-ok) specify in case verification is needed
      * @param prvkey (null-ok) specify in case signing is needed
+     * @deprecated Use {@link #initVerify(PublicKey)} or {@link #initSign(PrivateKey)} instead.
      */
+    @Deprecated
     void init(PublicKey pubkey, PrivateKey prvkey);
```

Comment:
I don't think anybody has implemented their own signatures. And even if they did, we broke the contract already by adding the new methods. let's just remove the cruft and only have the new methods.

Reply:
Sure thing.